tests/func-tests/virtual_machines_test.go

```diff
@@ -3,7 +3,10 @@ package tests_test
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/kubevirt/cluster-network-addons-operator/pkg/apis"
```

Comment:
the import sections are mixed here. we should have 4 groups:
1. standard golang library imports.
2. 3rd party librarys
3. test libraries
4. (optional) organization libraries (kubevirt)
5. self imports

like so: 
```
import (
	"encoding/json"
	"fmt"
	"time"

	corev1 "k8s.io/api/core/v1"
	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1" (notice a duplication here)
	"k8s.io/client-go/kubernetes/scheme"

	. "github.com/onsi/ginkgo"
	. "github.com/onsi/gomega"
	testscore "kubevirt.io/kubevirt/tests"

	"github.com/kubevirt/cluster-network-addons-operator/pkg/apis"
	networkaddonsv1 "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1"
	"kubevirt.io/kubevirt/tests/flags"
	kubevirtv1 "kubevirt.io/client-go/api/v1"
	"kubevirt.io/client-go/kubecli"

	tests "github.com/kubevirt/hyperconverged-cluster-operator/tests/func-tests"
)
```
you may need to fmt this.